### DISCUSSION TITLE
How to disable this top bar in the window?

### DISCUSSION THREAD

**Discussion by rendick** [i3/i3#5694] [Q&A] [answered]
![image](https://github.com/i3/i3/assets/91757099/d0235825-abbd-4803-869b-1e5be0c7777e)

**Comment by orestisfl** [comment, marked as answer]
See tracking issue: https://github.com/i3/i3/issues/3001

To remove this container, you just need to move the window to the left/right which will move it out of the surrounding container

**Reply by rendick** [reply]
Thanks a lot
